src/bfs/shared/PostScript.bf

```diff
@@ -0,0 +1,339 @@
+/*---------------------------------------------------------
```

Comment:
PostScript?!

Reply:
I'm not exactly sure what these are doing; I copied the datareader batch file and the shared folder from datamonkey-js